release_notes/Service_Fabric_ReleaseNotes_82CU2.md

```diff
@@ -0,0 +1,79 @@
+# Microsoft Azure Service Fabric 8.2 Second Refresh Release Notes
+
+This release includes the bug fixes described in this document. This release includes runtime, SDKs, and Windows Server Standalone deployments to run on-premises.
+
+
+The following packages and versions are part of this release:
+
+ | Service | Platform | Version | 
+ | --------- | ---------- | --------- | 
+ | Service Fabric Runtime | Ubuntu 18 <br> Windows | 8.2.1285.1 <br> 8.2.1486.9590 | 
+ | Service Fabric for Windows Server | Service Fabric Standalone Installer Package | 8.2.1486.9590 | 
+ | .NET SDK | Windows .NET SDK <br> Microsoft.ServiceFabric <br> Reliable Services and Reliable Actors <br> ASP.NET Core Service Fabric integration | 5.2.1363 <br> 8.2.1486 <br> 8.2.1486 <br> 8.2.1486 | 
+ | Java SDK | Java for Linux SDK | 1.0.6 | 
+ | Service Fabric PowerShell and CLI | AzureRM PowerShell Module <br> SFCTL | 0.3.15 <br> 11.1.0 | 
+
+
+## Contents 
+
+Microsoft Azure Service Fabric 8.2 Second Refresh Release Notes
+
+* [Current Breaking Changes and Updated Support](#Current-Breaking-Changes-And-Updated-Support)
+* [Service Fabric Common Bug Fixes](#Service-Fabric-Common-Bug-Fixes)
+* [Repositories and Download Links](#Repositories-And-Download-Links)
+
+
+## Current Breaking Changes and Updated Support
+* **Guest Executable and Container Applications**: Guest executable and container applications created or upgraded in Service Fabric clusters with runtime versions 7.1+ are incompatible with prior Service Fabric runtime versions (e.g. Service Fabric 7.0).<br/>
```

Comment:
Are Service Fabric 7.0 and 7.1 still considered supported?

Reply:
Unfortunately, we still have partners running on it. So, yes.